pkg/network/usm/ebpf_gotls_helpers.go

```diff
@@ -97,6 +97,10 @@ func (p *goTLSBinaryInspector) Inspect(fpath utils.FilePath, requests []uprobes.
 
 // Cleanup removes the inspection result for the binary at the given path from the map.
 func (p *goTLSBinaryInspector) Cleanup(fpath utils.FilePath) {
+	if p.offsetsDataMap == nil {
+		log.Warnf("offsetsDataMap is nil, cannot remove inspection result")
+	}
```

Comment:
1. shouldn't we return here?
2. Use `Warn` instead of `Warnf`

Reply:
Fixed!